src/ConnectionCard/ConnectionCard.js

```diff
@@ -0,0 +1,102 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { Badge } from '../Badge';
+import { Price } from '../Price';
+import ConnectionCardRow from './ConnectionCardRow';
+import TripSector from './TripSector';
+import BadgesContainer from './BadgesContainer';
+import type { BadgeProps } from '../Badge/BadgeTypes';
+import type { PriceProps } from '../Price/PriceTypes';
+import type { TripSectorProps } from './ConnectionCardTypes';
+import StyleSheet from '../PlatformStyleSheet';
+
+type TripSectorWithId = { ...TripSectorProps, id: number };
+type BadgeWithId = { ...BadgeProps, id: number };
+
+type Props = {|
+  padding?: boolean,
+  wayForth: Array<TripSectorWithId>,
+  wayBack?: Array<TripSectorWithId>,
+  price: PriceProps,
+  duration?: string,
+  badges?: Array<BadgeWithId>,
+|};
+
+const renderTripSector = way =>
+  way.map(connection => (
+    <TripSector
+      key={connection.id}
+      arrival={connection.arrival}
+      arrivalTime={connection.arrivalTime}
+      carrier={connection.carrier}
+      tripDate={connection.tripDate}
+      departure={connection.departure}
+      departureTime={connection.departureTime}
+      duration={connection.duration}
+    />
+  ));
+
+const renderBadges = (badges: Array<BadgeWithId>) =>
```

Comment:
We don't need this prams type notation, because `renderBadges ` is not exported so flow is able to check this types without notation, but if this function has been exported you must add the notation to param.